src/components/APIDoc/SchemaDataView.tsx

```diff
@@ -136,8 +136,10 @@ const getTreeViewData = (schemaName: string, schema: DeRefResponse<OpenAPIV3.Arr
     if (conditionalSchema) {
       const {schemaKey, schemaVal} = conditionalSchema
       children = [{name: <ConditionSchema condition={schemaKey} schemas={schemaVal} document={document}/>}] as TreeViewDataItem[]
-    } else if ('type' in value) {
-      propertyType = value.type as string
+    } else if ('$ref' in value ) {
+      propertyType = value.$ref.split('/').at(-1) as string
+    }  else {
+      propertyType = 'type' in value ? value.type as string : 'any/undefined type'
```

Comment:
The undefined json schema type means that anything is possible - I guess it would be wrong to use `undefined` which in js/json means "value is not set"
```suggestion
      propertyType = 'type' in value ? value.type as string : 'any type'
```
or maybe `Unrestricted type`

Reply:
Makes sense :+1: